roles/client_side_tests/tasks/get_prom_info.yml

```diff
@@ -1,14 +1,14 @@
 ---
-- name: "Get the default-prometheus-htpasswd secret"
+- name: "Generate an access token for prometheus"
   ansible.builtin.shell:
     cmd: |
-      oc  get secret default-prometheus-htpasswd -ojson | jq '.data.auth, .data.password' |  sed 's/"//g'
-  register: prom_secret
+      oc create token stf-prometheus-reader
```

Comment:
For backwards-compatibility, I suggest keeping the "get password" and "get user" tasks, and stick them in a block, guarded with a `when: X`.

The new token generation can be in a complementary block (i.e. `when: not X`)

To keep the curl tasks clean, I suggest adding a "prom_auth_string" that would be either `-u "{{ prom_user }}:{{ prom_pass }}"` or `-u "{{ prom_user }}:{{ prom_pass }}"`